tests/report_tests/plugin_tests/system.py

```diff
@@ -0,0 +1,70 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos_tests import StageOneReportTest, StageTwoReportTest
+
+
+class SystemPluginTest(StageOneReportTest):
+    """Basic sanity check to make sure common config files are collected
+
+    :avocado: tags=stageone
+    """
+
+    sos_cmd = '-o system'
+
+    def test_system_files_collected(self):
+        self.assertFileCollected("/proc/sys")
+        self.assertFileCollected("/etc/sysconfig")
+        self.assertFileCollected("/etc/default")
+        self.assertFileCollected("/etc/environment")
+
+    def test_system_files_forbidden(self):
+        self.assertFileNotCollected("/proc/sys/net/ipv4/route/flush")
+        self.assertFileNotCollected("/proc/sys/net/ipv6/route/flush")
+        self.assertFileNotCollected("/proc/sys/net/ipv6/neigh/*/retrans_time")
+        self.assertFileNotCollected("/proc/sys/net/ipv6/neigh/*/" +
+                                    "base_reachable_time")
+        self.assertFileNotCollected("/etc/default/grub.d/" +
+                                    "50-curtin-settings.cfg")
+
+    def test_system_cmd_collected(self):
+        self.assertFileCollected("ld.so_--help")
+        self.assertFileCollected("ld.so_--list-diagnostics")
+        self.assertFileCollected("ld.so_--list-tunables")
+
+
+class SystemScrubTest(StageTwoReportTest):
+    """Ensure that environment, default and sysconfig are picked up
+    and properly scrubbed
+
+    :avocado: tags=stagetwo,scrub
+    """
+    sos_cmd = '-o system'
+    files = [
+        ('../../../tests/test_data/system_test_data', '/etc/environment'),
+        ('../../../tests/test_data/system_test_data', '/etc/default/proxy'),
```

Comment:
When I run `PYTHONPATH=tests/ avocado run -p TESTLOCAL=true --test-runner=runner -t stagetwo tests/report_tests` on my test rhel8 system, I get:

     (17/57) tests/report_tests/plugin_tests/system.py:SystemScrubTest.test_archive_created: ERROR: [Errno 2] No such file or directory: '/etc/default/proxy' (0.00 s)

Did I run the avocado test in a proper way..? (I am often uncertain on this..) FYI my system has no `/etc/default/proxy` file/dir.

Reply:
I cannot find that file/dir either in Fedora or RHEL 9. What package creates it?